frontend/e-buddy/src/components/landing/Footer.tsx

```
@@ -0,0 +1,250 @@
+            </div>
+
+            <div className="flex items-center space-x-6">
+              <a
+                href="#"
+                className="text-gray-400 hover:text-white transform hover:scale-110 transition-all duration-300"
+              >
+                <Youtube className="h-5 w-5" />
+                <span className="sr-only">Youtube</span>
+              </a>
+              <a
+                href="#"
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)